dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/TestEnableDisableFeaturesTest.java

```diff
@@ -389,7 +389,8 @@ public void testEDF12() throws Exception {
     	serverEDF12.startServer();
     	Log.info(c, testName, "------- Remove monitor-1.0 ------");
     	serverEDF12.setServerConfigurationFile("server_noJDBCMonitor.xml");
-    	Assert.assertNotNull("CWPMI2002I NOT FOUND",serverEDF12.waitForStringInLogUsingMark("CWPMI2002I"));
+    	Assert.assertNotNull("CWWKF0013I NOT FOUND",serverEDF12.waitForStringInLogUsingMark("CWWKF0013I"));
+    	Assert.assertNotNull("CWWKF0008I NOT FOUND",serverEDF12.waitForStringInLogUsingMark("A CWWKF0008I"));
```

Comment:
Is `CWWKF0008I` an audit message?

Reply:
I would suggest to use `setMarkToEndOfLog()` before `setServerConfigurationFile()` so you just need to wait for `CWWKF0008I`.